.github/workflows/build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         source actions-ci/install.sh
     - name: Pip install pylint, black, & Sphinx
       run: |
-        pip install --force-reinstall pylint==1.9.2 black==19.10b0 Sphinx sphinx-rtd-theme
+        pip install pylint black==19.10b0 Sphinx sphinx-rtd-theme
```

Comment:
I would leave the `--force-reinstall` here. As it currently stands, `actions-ci/install.sh` still pip installs pylint and Sphinx+theme. If you look at this step in the passing job, it shows that it skips the install and uses pylint 1.9.2.

Reply:
@kattni ^